packages/p2p/src/pages/app.jsx

```diff
@@ -272,8 +275,13 @@ const App = () => {
     }, [action_param, code_param]);
 
     // TODO: This will redirect the internal users to the standalone application temporarily. Remove this once the standalone application is ready.
-    if (is_p2p_v2_enabled) window.location.href = URLConstants.derivP2pProduction;
```

Comment:
can we also remove `p2p_v2` from `feature_flags` store?

Reply:
done